cpp/test/Glacier2/staticFiltering/test.py

```diff
@@ -298,4 +298,6 @@ def setup(self, current):
         for testcase in testcases:
             self.addTestCase(Glacier2StaticFilteringTestCase(testcase, hostname))
 
-Glacier2StaticFilteringTestSuite(__name__, testcases=[], runOnMainThread=True, options={ "ipv6" : [False] }, multihost=False)
+
+Glacier2StaticFilteringTestSuite(__name__, testcases=[], runOnMainThread=True,
+                                 options={"ipv6": [False]}, multihost=False)
```

Comment:
Is this configurable? I prefer one parameter per line when we have to split a long line.

Reply:
For now I think we should just to the defaults. Once all the linting is done we can go back and tweak the style if we really want to.